source/server/server_configuration_parser.h

```diff
@@ -19,7 +21,7 @@ static const char* kInvalidExePathMessage = "The server was unable to resolve th
 static const char* kInvalidMaxMessageSizeMessage = "The max message size must be an integer.";
 static const char* kInvalidFeatureToggleMessage = "Feature Toggles must be specified as boolean fields in the form \"feature_toggles\": { \"feature1\": true, \"feature2\": false }. \n\n";
 static const char* kInvalidCodeReadinessMessage = "code_readiness must be a string in [Release, RestrictedRelease, NextRelease, RestrictedNextRelease, Incomplete, Prototype].\n\n";
-static const char* kDefaultAddressPrefix = "[::]:";
+static const char* kDefaultAddressPrefix = "[::]";
```

Comment:
This is not really a prefix anymore, so I think it makes sense to drop "Prefix" from the constant name.

Reply:
Renamed to kDefaultAddress